### ISSUE TITLE
Rendering Errors with ItemsRepeater Inside StackPanel on Android Platform

### ISSUE THREAD

**Issue by LITTOMA** [unoplatform/uno#14583]
### Current behavior

Hello,

I've encountered an issue where placing an ItemsRepeater inside a StackPanel causes unexpected rendering errors. The items within the ItemsRepeater don't render correctly.

https://github.com/unoplatform/uno/assets/10324313/cdd4442b-41df-4122-bb72-c34e4ee23b85

### Expected behavior

Items of ItemsRepeater renered correctly on Android.

### How to reproduce it (as minimally and precisely as possible)

Please check the sample [here](https://github.com/LITTOMA/ItemsRepeaterTest)

### Workaround

N/A

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### Affected platforms

Android

### IDE

Visual Studio 2022

**Comment by limeniye** [comment]
Hello, @LITTOMA.
Why do you need to put ItemsRepeater to the StackPanel?

**Comment by LITTOMA** [comment]
> Hello, @LITTOMA. Why do you need to put ItemsRepeater to the StackPanel?

@limeniye I need to put some other elements into the StackPanel along with the ItemsRepeater. Then put the StackPanel into a ScrollViewer. That's pretty common, isn't it?

**Comment by limeniye** [comment]
In WinUI it looks the same
![image](https://github.com/unoplatform/uno/assets/69314237/370428e0-2c1b-4870-bd03-d12ee343794b)

**Comment by jeromelaban** [comment]
Putting an `ItemsRepeater` inside of a `StackPanel` is not particularly common, no. `ItemsRepeater` is used in scenarios where layout is generally complex and virtualized. It is likely that there is a particular condition related to the way StackPanel measures its children.

You may want to try to use an `ItemsControl` instead, and use the header/footer properties, if this is the scenario that you're looking for.

**Comment by jeromelaban** [comment]
Indeed, this is definitely an issue, the layout is not supposed to change.

**Comment by MartinZikmund** [comment]
Specific to Android native, should work correctly in Skia Android
